app/components/requests-item.js

```diff
@@ -37,6 +37,10 @@ export default Ember.Component.extend({
   message: function() {
     var message;
     message = this.get('request.message');
+
+    if (message === 'github pages branch') {
```

Comment:
I think it would be better to create a property `isGHPages`, which depends on `message`. Setting other properties inside other properties is generally discouraged as in such case you depend on `message` being called to have `isGHPages` correctly set.